frontend/app/src/pages/assets/[identifier].vue

```diff
@@ -106,6 +106,22 @@ const collectionBalance = computed<AssetBalanceWithPrice[]>(() => {
   return get(aggregatedBalances).find(data => data.asset === get(identifier))?.breakdown || [];
 });
 
+const collectionAssetWithPrice = computed<string | undefined>(() => {
+  const collectionBalanceVal = get(collectionBalance);
+
+  const id = get(identifier);
+
+  if (collectionBalanceVal.length === 0) {
+    return id;
+  }
+
+  if (collectionBalanceVal.some(item => item.asset === id)) {
+    return id;
+  }
+
+  return collectionBalanceVal[0].asset;
+});
+
```

Comment:
This part here to fix the issue if we show the graph for a collection.
You see if we do it for collection, it will use the price from the `mainAsset` from that collection.
But there is a case where the user doesn't have the `mainAsset` in their balance, hence the price is not queried (0). We can use other asset in that collection that the user own, and use the price.